helmtestserver/server_test.go

```diff
@@ -0,0 +1,65 @@
+package helmtestserver
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"golang.org/x/crypto/openpgp"
+	"helm.sh/helm/v3/pkg/downloader"
+)
+
+func TestPackageChart(t *testing.T) {
+	server, err := NewTempHelmServer()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(server.Root())
+	publicKeyPath := fmt.Sprintf("%s/%s", server.Root(), "pub.pgp")
+	packagedChartPath := fmt.Sprintf("%s/%s", server.Root(), "helmchart-0.1.0.tgz")
```

Comment:
Also, let's use `filepath.Join()` here for constructing paths.

Reply:
> easy/automatic cleanup with the root.
Yes, that's the reason for creating these files within the server root. I don't see how it matters to keep these files in a separate temp dir, since they'll all be cleaned at the end of the test anyway.